### PR TITLE
Add tag action and reducer

### PR DESCRIPTION
**Pull request by amandahogan**

### CODE REVIEW

File: src/actions/tags.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+let nextTagId = 6
+
+export const addTag = (categoryId, value) => {
+  return {
+    payload: {
+      id: nextTagId++,
+      categoryId: categoryId,
+      value: value
```
Comment (Quinn-H) [inline src/actions/tags.js]: when object property name  and value have same name, you can simplify it by only write property name. like this 
`payload: {
         id: nextTagId++,
         categoryId,
         value
}`

Comment (davidchristie) [inline src/actions/tags.js]: Please change to:
`   categoryId,
    value
`

File: src/reducers/tags.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  {
+    id: 5,
+    categoryId: 3,
+    value: 'AngluarJS',
+    companies: [
+      {companyId: 0}
+    ]
+  }
+]
```
Comment (davidchristie) [inline src/reducers/tags.js]: Looks good! Please change the initial state `companies` to match these tags.
I.e. Company id=0 will start with a `tags` array equal to `[0]`

**Review by davidchristie — CHANGES_REQUESTED** [review]
The `companies` reducer needs to be modified as well so that whenever a tag is added it gets added to the `tags` property of the specified company.

**Review by Quinn-H — APPROVED** [review]

**Review by davidchristie — APPROVED** [review]

**Pull request merged**